### PR TITLE
feat: fastify eject

### PR DESCRIPTION
**Pull request by joobisb**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

This PR adds support for fastify eject as discussed in #277

### CODE REVIEW

File: templates/eject/server.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+// Read the .env file.
+require('dotenv').config()
+
+// Require the framework
+const Fastify = require('fastify')
+
+// Instantiate Fastify with some config
+const app = Fastify({
+  logger: true,
+  pluginTimeout: 10000
```
Comment (mcollina) [inline templates/eject/server.js:12]: This is the default, we can remove

File: help/eject.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Usage: fastify eject
+
+Turns your application into a standalone executable with a server.js file being added.
```
Comment (mcollina) [inline help/eject.txt:3]: Can you document the PORT env variable?

**Review by mcollina — CHANGES_REQUESTED** [review]

File: templates/eject/server.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+})
+
+// Register your application as a normal plugin.
+app.register(require('./app.js'))
```
Comment (zekth) [inline templates/eject/server.js:16]: i'd personally have a proper statement:
`const app_service = require('./app.js')`

File: help/eject.txt (hunk 2 above)
Comment (joobisb) [reply help/eject.txt:3]: sure

File: templates/eject/server.js (hunk 3 above)
Comment (joobisb) [reply templates/eject/server.js:16]: okay. i'll update

**Comment by joobisb** [comment]
@mcollina @zekth i've updated the PR. please have a look

**Review by zekth — APPROVED** [review]
lgtm

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**